### DISCUSSION TITLE
Resampling funcion

### DISCUSSION THREAD

**Discussion by MariaICV** [h2020charisma/oranchada#41] [Q&A]
I have some problems to use the resampling function.
I am using the workflow that you give us in the Sofia Meeting:
![image](https://github.com/h2020charisma/oranchada/assets/129367889/d0380536-e3ee-4de3-9260-1e65de43646c)
When  I load the Neon files that you indicate, I can do it without problems. When I want to use another spectra (for example, S0B), first I can see the correct spectra:
![image](https://github.com/h2020charisma/oranchada/assets/129367889/4e637af0-ad17-467e-ae32-59c9c832bdbd)
But however, when I try to obtain the final results, I obtain the spectra of the Neon again. The problem is in the step of Merger after the baseline suctrations as you can see here:
![image](https://github.com/h2020charisma/oranchada/assets/129367889/ceac4f92-5c19-4d05-902a-5e2b68855ea4)

Then, when I want to try this function with my reference sample spectra, it seems that it works (It does not happen as in the case of S0B, I can see the corresponding spectra of the reference sample):
![image](https://github.com/h2020charisma/oranchada/assets/129367889/45ad5a21-2c04-4df6-96e9-62066200e45b)
but when I want to obtain the final spectra, there is a Raman shift of both spectra that it should not happen: 
![image](https://github.com/h2020charisma/oranchada/assets/129367889/603929cd-23b0-40c9-abb1-d2650bda94aa)

Thank you!

**Comment by georgievgeorgi** [comment]
Could you please make sure that `x-min` and `x-max` parameters of all `Resample NUDFT` are equal wider than all spectra?
Ex: if the x-axis of a spectrum starts at -100, the `x-min` of all widgets should be smaller than -100, say -101.
Also it seems that you are using older version. So please try to update.
Let me know if the problem still persist.

**Comment by MariaICV** [comment]
Thank you very much Georgi!! it works! The problem was the x-min effectively.

Thank you again.

María
